src/Compilers/Core/Portable/Operations/ControlFlowGraphBuilder.cs

```diff
@@ -5682,7 +5711,7 @@ public override IOperation VisitObjectCreation(IObjectCreationOperation operatio
         {
             EvalStackFrame frame = PushStackFrame();
             EvalStackFrame argumentsFrame = PushStackFrame();
-            ImmutableArray<IArgumentOperation> visitedArgs = VisitArguments(operation.Arguments);
```

Comment:
>VisitArguments [](http://example.com/codeflow?start=61&length=14)

It is not clear why we simply didn't pass ```false``` as the second argument.

Reply:
Because the first approach needed this version of the call, and I didn't fully revert this bit when I took the new approach.